packages/service-library/tests/with_postgres/test_aiopg_utils.py

```diff
@@ -0,0 +1,158 @@
+# pylint:disable=unused-argument
+# pylint:disable=redefined-outer-name
+# pylint: disable=too-many-arguments
+import sys
+from copy import deepcopy
+from pathlib import Path
+
+import pytest
+import sqlalchemy as sa
+import yaml
+from aiohttp import web
+from aiopg.sa import Engine, create_engine
+
+from servicelib.aiopg_utils import (ATTEMPTS_COUNT, DatabaseError,
+                                    DataSourceName, is_postgres_responsive,
+                                    retry_pg_api)
+
+current_dir = Path(sys.argv[0] if __name__ == "__main__" else __file__).resolve().parent
+
+
+metadata = sa.MetaData()
+tbl = sa.Table('tbl', metadata,
+               sa.Column('id', sa.Integer, primary_key=True),
+               sa.Column('val', sa.String(255)))
+
+# FIXTURES ------------
+
+@pytest.fixture(scope='session')
+def docker_compose_file() -> Path:
+    # overrides fixture from https://github.com/AndreLouisCaron/pytest-docker
+    return current_dir / 'docker-compose.yml'
```

Comment:
isn't that already in conftest? why put it there?
this will not restart the docker-compose up call

Reply:
no, it is not. Could it be that you are getting confused with the webserver tests? Note that this is servicelib package!